styles/components/_buttons.scss

```diff
@@ -325,6 +325,30 @@
   }
 }
 
+.btnFlx {
+  // see styleSheet.html for example usage.
+  // use for buttons in the new 'flex' style with more padding and use css flex.
+  @extend .btn;
+  padding: 25px;
+
+  border-radius: 0px;
+  border-top: 1px solid;
+
+  & + a {
+    border-left: 1px solid #dbdbdb; // Move this color into themes.
+  }
+
+  + a:hover {
+    margin-right: -1px;
+  }
+
+  &:hover, &:focus {
+    z-index: 0;
+    border: 1px solid;
+    margin: 0px -2px -1px -1px;
```

Comment:
Playing with margins here to get the hover/focus borders to line up well with the existing container. There's still a little content shifting going on, so I'm open to thoughts on how to better do this.

Reply:
I found a better way to prevent the shifting.

Set the borders to transparent, then set the top to be a color.

That way the border is always there, it's just not visible unless it's being hovered over.